go-controller/pkg/config/config.go

```diff
@@ -1048,12 +1055,28 @@ var K8sFlags = []cli.Flag{
 		Usage:       "absolute path to the Kubernetes kubeconfig file (not required if the --k8s-apiserver, --k8s-ca-cert, and --k8s-token are given)",
 		Destination: &cliConfig.Kubernetes.Kubeconfig,
 	},
+	&cli.StringFlag{
+		Name:        "bootstrap-kubeconfig",
+		Usage:       "absolute path to the Kubernetes kubeconfig file that is used to create the initial client certificates(not required if the k8s-kubeconfig is given)",
```

Comment:
I'm ok with that. 

Additionally

should `bootstrap-kubeconfig` talk about `the node identity certificates` specifically instead of just `the initial client certificates`? Given that the CSRs that are built are tailored to the identity binary to approve, a user shouldn't expect the certificates to be bootstrapped just by providing this option, it needs some other piece such that this isn't just a generic bootstrap. This should also imply this option is specific to node components.


I think the mentions to `k8s-apiserver` need to be tweaked, as it is only an alternative when it is an http endpoint and not https. Does not need to happen in this PR.